woodwork/schema_column.py

```diff
@@ -13,7 +13,7 @@
 def _get_column_dict(name,
                      logical_type,
                      semantic_tags=None,
-                     use_standard_tags=True,
+                     use_standard_tags=False,
```

Comment:
We should also change the docstring which has `If True, will add standard semantic tags to the column based
                specified logical type. Defaults to True.`

Reply:
> If True, will add standard semantic tags to the column based specified logical type

@tamargrey should this be "to the column based on the specified logical type"?

Reply:
yes that definitely is a lot more clear! (The same typo is on the table Schema docstring and the TableAccessor docstring as well, I just noticed 🤦‍♀️ )